src/scss/attendee/theme.scss

```
@@ -2,18 +2,54 @@
 // Please make sure their names have semantic meaning
 // i.e. instead of $white it would be $c-button--fg or $c-input--bg
 
-$sparkle--bg: rgb(99, 42, 199); // previously named $sparkle-theme
-$sparkle--border: rgba(130, 130, 130, 0.3); // previously named $sparkle-border
+// All the colour definitions are dona via CSS variables and then we use SCSS
+// variables to substitue them into the right place.
+:root {
+  --default-text: rgb(0, 0, 0);
+  --chat-input-background: rgb(230, 230, 230);
+
+  --panel-background: rgb(255, 255, 255);
+  --glass-panel-background: rgba(255, 255, 255, 0.4);
+
+  --sparkle--bg: rgb(99, 42, 199);
+  --sparkle--border: rgba(130, 130, 130, 0.3);
+
+  --button-fg: rgb(255, 255, 255);
+  --button-bg: rgb(99, 42, 199);
+
+  --overlay-fg: rgb(255, 255, 255);
+  --overlay-bg: rgba(99, 42, 199, 0.7);
```

This might make it obvious to us if we're missing some dark or light version as there will be no assumptions of defaults like bellow

```suggestion

@media (prefers-color-scheme: light) {
  --default-text: rgb(0, 0, 0);
  --chat-input-background: rgb(230, 230, 230);
  
  --panel-background: rgb(255, 255, 255);
  --glass-panel-background: rgba(255, 255, 255, 0.4);
  
  --sparkle--bg: rgb(99, 42, 199);
  --sparkle--border: rgba(130, 130, 130, 0.3);
  
  --button-fg: rgb(255, 255, 255);
  --button-bg: rgb(99, 42, 199);
  
  --overlay-fg: rgb(255, 255, 255);
  --overlay-bg: rgba(99, 42, 199, 0.7);
 }
```